test/end-to-end/pages/screeners/willoriginal.js

```diff
@@ -1,17 +1,23 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/screeners/willoriginal');
+const willOriginalEn = require('app/resources/en/translation/screeners/willoriginal');
+const willOriginalCy = require('app/resources/cy/translation/screeners/willoriginal');
+const contentEn = require('app/resources/en/translation/common');
+const contentCy = require('app/resources/cy/translation/common');
+const pageUnderTest = require('app/steps/ui/screeners/willoriginal');
```

Comment:
This is undoing performance work - including these files takes time - do we need to?